### ISSUE TITLE
tortoisesvn is not build for newer v1.14.6++ due appveyor failure 'Chocolatey-AU' was not loaded

### ISSUE THREAD

**Issue by ThomasPatzig** [chocolatey-community/chocolatey-packages#2542]
### Chocolatey Version

2.3.0

### Chocolatey License

None

### Package Version

1.14.5.29465

### Current Behaviour

see appveyor build error:

The specified module 'Chocolatey-AU' was not loaded because no valid module file was found in any module directory.
    + CategoryInfo          : ResourceUnavailable: (Chocolatey-AU:String) [Import-Module], FileNotFoundException
    + FullyQualifiedErrorId : Modules_ModuleNotFound,Microsoft.PowerShell.Commands.ImportModuleCommand
    + PSComputerName        : localhost
 
   [229/230] wesnoth is updated to 1.16.11.20240311 (134.83s)

https://ci.appveyor.com/project/chocolateycommunity/chocolatey-packages/builds/49373520

### Steps To Reproduce

https://ci.appveyor.com/project/chocolateycommunity/chocolatey-packages/builds/49373520

### Environment

```markdown
appveyor
```

### Chocolatey Log

```shell
The specified module 'Chocolatey-AU' was not loaded because no valid module file was found in any module directory.
    + CategoryInfo          : ResourceUnavailable: (Chocolatey-AU:String) [Import-Module], FileNotFoundException
    + FullyQualifiedErrorId : Modules_ModuleNotFound,Microsoft.PowerShell.Commands.ImportModuleCommand
    + PSComputerName        : localhost
 
   [229/230] wesnoth is updated to 1.16.11.20240311 (134.83s)
The specified module 'Chocolatey-AU' was not loaded because no valid module file was found in any module directory.
```

**Comment by ThomasPatzig** [comment]
https://tortoisesvn.net/Changelog.txt

latest versions of packages missing

Version 1.14.7
Version 1.14.6

**Comment by fbuehler-vector** [comment]
should be resolved via #2446
